snippets/product/content-course.liquid

```diff
@@ -13,7 +13,7 @@
         data-translation-release-at="{{ 'lesson.release_at' | t }}"
         data-translation-release-after="{{ 'lesson.release_after' | t }}"
         data-translation-course_content-views="{{ 'product.course_content.views' | t }}"
-        data-api-key="{{ current_apps_credentials.course_content }}">
+        data-api-key="{% if current_api_key %}{{current_api_key.credentials}}{% else %}{{ current_apps_credentials.course_content }}{% endif %}">
```

Comment:
@maykefreitas se caso for pra tirar, uma alteração que fiz no core, eu volto este api_key pro jeito que tava. então caso aprove não mergeia de uma vez não.